frontend/src/components/modalContent/AddIncidentModal.js

```diff
@@ -153,11 +182,17 @@ const StepForms = (props, next, prev, handleIncidentSubmit) => [
 ];
 
 export class AddIncidentModal extends Component {
-  state = { current: 0 };
+  state = {
+    current: 0,
+    uploadedFiles: this.props.initialValues.documents
+      ? [...this.props.initialValues.documents]
+      : [],
+  };
 
   handleIncidentSubmit = () => {
     this.props.onSubmit({
       ...this.props.addIncidentFormValues,
+      updated_documents: this.state.uploadedFiles,
     });
     // TODO: Catch error
     this.close();
```

Comment:
Are the documents orphaned if they cancel upload? I know we talked about running a cron to deal with orphaned documents.

Reply:
Documents are orphaned, yes. Orphaned documents are a system concern and occur in many places, and we really do need to deal with them.